src/test/java/core/basesyntax/DataConverterImplTest.java

```diff
@@ -0,0 +1,97 @@
+package core.basesyntax;
+
+import core.basesyntax.converter.DataConverter;
+import core.basesyntax.converter.DataConverterImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;

```

Comment:
Use proper class for assertions:
```suggestion
import org.junit.jupiter.api.Assertions;

```